.bingo/Variables.mk

```diff
@@ -29,6 +29,12 @@ $(CONTROLLER_GEN): $(BINGO_DIR)/controller-gen.mod
 	@echo "(re)installing $(GOBIN)/controller-gen-v0.16.1"
 	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=controller-gen.mod -o=$(GOBIN)/controller-gen-v0.16.1 "sigs.k8s.io/controller-tools/cmd/controller-gen"
 
+CRD_DIFF := $(GOBIN)/crd-diff-v0.0.0-20241112183958-25304aa59cdb
+$(CRD_DIFF): $(BINGO_DIR)/crd-diff.mod
+	@# Install binary/ries using Go 1.14+ build command. This is using bwplotka/bingo-controlled, separate go module with pinned dependencies.
+	@echo "(re)installing $(GOBIN)/crd-diff-v0.0.0-20241112183958-25304aa59cdb"
+	@cd $(BINGO_DIR) && GOWORK=off $(GO) build -mod=mod -modfile=crd-diff.mod -o=$(GOBIN)/crd-diff-v0.0.0-20241112183958-25304aa59cdb "github.com/everettraven/crd-diff"
```

Comment:
Should this be in an `operator-framework` org repo? Or maybe just all the code under `hack/` and get it built from there?

Reply:
We use tools like this from non operator-framework repos already.

I'm not opposed to donating it to the operator-framework, but we could update this whenever that happens. Using as is gets us these checks in sooner rather than later.

Reply:
Chatted a bit with folks at KubeCon NA 2024 and there seems to be some interest in bringing something like this up to the sig-apimachinery folks to consolidate some of the various CRD validation efforts that have been going on into a single concerted effort. I'm planning to get involved in that area in the hopes we can get some official, Kubernetes sanctioned, tooling in place. 

That said, I still think it is worth it to implement the check as-is since it provides some immediate value.